TESTING.md

```diff
@@ -151,10 +151,6 @@ following values:
 - `couchdb-master`: use this value if you already have CouchDB running; it
   causes `COUCH_HOST` to be set to the correct value.
 
-#### `SKIP_MIGRATION` (default: `0`)
```

Comment:
why is this removed?

Reply:
This PR is in draft; reviewing it may be confusing at this time.